common/rabbitmq/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: rabbitmq
-version: 0.6.12
+version: 0.6.13
```

Comment:
Please bump the version to `0.7.0` because the PR introduces a new feature and not just a bugfix